javascript/javascript3/week2/Movies/movies_array.js

```diff
@@ -0,0 +1,39 @@
+//1. Create an array of bad movies
+const apiUrl = `https://tinyurl.com/y9zck5xx`;
+// Bad movies rated for <=4
+async function badMoviesArray() {
+    //a.fetching the movies data
+    fetch(apiUrl)
+        .then((response) => response.json())
+        .then((movies) => {
+            //b.filtering bad movies accrotding to ratings
+            const badMovies = movies
+                .filter((movie) => movie.rating <= 4)
+                .map((movie) => movie);
+            console.log(badMovies);
+        })
+        .catch((error) => {
+            //b.Throw an error if above condition failed
+            console.log(error);
+        });
+}
+badMoviesArray();
+
+//2. Create an array of bad movies since year 2000
+
+const badMoviesSince2000 = () => {
```

Comment:
When we call badMoviesSince2000 we have no way of detecting when the fetch is resolved. 
Its better to return a promise